package.json

```diff
@@ -16,6 +16,7 @@
 		"build-win": "if exist _site (rd /s /q _site; && bundle exec jekyll build) else (bundle exec jekyll build)",
 		"federalist": "webpack && npm run release-styleguide",
 		"start-states": "bundle exec jekyll serve --incremental --config _config.yml,_config-dev.yml",
+		"start-jekyll": "bundle exec jekyll build --watch --incremental --config _config.yml,_config-dev.yml",
```

Comment:
Is this related to the withheld caption issue, @master12? You may want to call this change out in the summary list items if separate.